src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -0,0 +1,575 @@
+--!strict
+
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local MarketplaceService = game:GetService("MarketplaceService")
+
+local Packages = ReplicatedStorage.Packages
+local Knit = require(Packages.Knit)
+local GameAnalytics = require(Packages.GameAnalytics)
+
+local CURRENCIES = { "Coins" } -- List of all in-game currencies
+local BUILD = "0.0.1" -- Game version
+local GAME_KEY = "xxxxxxxxxxxxxxxx"
+local SECRET_KEY = "xxxxxxxxxxxxxxxx"
+local LOG_DEV_PRODUCT_PURCHASES = true
+local RESOURCE_EVENT_TYPES = { -- (example: player gained coins in a mission is a
+	-- "Reward" event type, player purchasing coins with Robux is a "Purchase" event type)
+	"Reward",
+	"Purchase",
+	"Shop",
+	"Loot",
+	"Combat"
+}
+local GAMEPASS_IDS = {} -- List of all gamepass ids in the game
+local CUSTOM_DIMENSIONS = {
+	-- Uncomment and fill each dimension as needed
+	-- Refer to https://docs.gameanalytics.com/advanced-tracking/custom-dimensions
+	-- for more information about dimensions
+	
+	-- DIMENSION_01 = {};
+	-- DIMENSION_02 = {};
+	-- DIMENSION_03 = {};
+}
+
+--[=[
+	@class AnalyticsService
+	@server
```

Comment:
I'm not sure if the realm should be defined w/Moonwave as 'server', because the client has access to functions under this service.

I'm personally of the view that the realm tag should be added if it's patently for use on the server/client, or the realm tag could be defined specifically per function if it's a mix of client/server functions.